elements/lederer-schoberl.def

```diff
@@ -0,0 +1,42 @@
+name: Lederer-Schoberl
+html-name: Lederer&ndash;Sch&ouml;berl
+min-degree: 0
+polynomial-subdegree: k
+polynomial-superdegree: k
+lagrange-subdegree: k
+lagrange-superdegree: k
+degree: polynomial-subdegree
+categories:
+  - matrix
+reference-elements:
+  - triangle
+ndofs:
+  triangle:
+    formula: 4(k+1)(k+2)/2
+    oeis: A045943
+  tetrahedron:
+    formula: 9(k+1)(k+2)(k+3)/6
+polynomial-set: poly[k]^dd
+dofs:
+  edges: integral moments of inner products of tangent and normal to edge with (lagrange,k)
+  faces:
+    - integral moments matrix trace with (lagrange,k-1)
+    - integrals of tensor products with bubbles with zero normal-tangent trace
```

Comment:
This does not correctly describe the element in 3D. There are two types of DOFs: on facets and on the cell interior.

Reply:
I've updated `dofs` and think they're now correct for both 2D and 3D